db/migrate/20160324203119_add_id_primary_key_to_join_tables.rb

```diff
@@ -0,0 +1,73 @@
+require_relative './20160106214719_add_composite_primary_keys_to_join_tables'
+
+class AddIdPrimaryKeyToJoinTables < ActiveRecord::Migration[5.0]
+  JOIN_TABLES = %w(
+    conditions_miq_policies
+    key_pairs_vms
+    miq_roles_features
+    miq_servers_product_updates
+    security_groups_vms
+    storages_vms_and_templates
+    miq_groups_users
+    cloud_tenants_vms
+    customization_scripts_operating_system_flavors
+    configuration_locations_configuration_profiles
+    configuration_organizations_configuration_profiles
+    configuration_profiles_configuration_tags
+    configuration_tags_configured_systems
+    container_groups_container_services
+    direct_configuration_profiles_configuration_tags
+    direct_configuration_tags_configured_systems
+    network_ports_security_groups
+  ).freeze
+
+  class MiqRegion < ApplicationRecord
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    say_with_time("Removing composite primary keys from join tables") do
+      revert AddCompositePrimaryKeysToJoinTables
+    end
+
+    JOIN_TABLES.each do |t|
+      execute "DELETE FROM #{t}" if on_replication_target?
```

Comment:
@carbonin Per our discussion, please add some logic to exclude rows from the global region from deletion.